armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -1197,20 +1197,29 @@ def readMovesYaml(fname):
                 )
 
             for action in actions:
-                allowed = {"cascade", "fuelEnrichment", "extraRotations", "misloadSwap"}
+                allowed = {"cascade", "fuelEnrichment", "extraRotations", "misloadSwap", "assemblyName"}
                 unknown = set(action) - allowed
                 if unknown:
                     raise InputError(f"Unknown action keys {unknown} in shuffle YAML")
 
                 if "cascade" in action:
                     chain = list(action["cascade"])
                     if len(chain) < 2:
-                        raise InputError("cascade must contain an assembly type and at least one location")
+                        raise InputError("cascade must contain at least two entries")
                     if any(not isinstance(item, str) for item in chain):
                         raise InputError("cascade entries must be strings")
 
-                    assemType = chain[0]
-                    locs = chain[1:]
+                    if chain[0] == "SFP":
+                        assemType = None
+                        locs = chain
+                        if len(locs) < 2:
+                            raise InputError("cascade starting with SFP must include a destination location")
+                    else:
```

Comment:
```suggestion
                    if chain[0] == "SFP":
                        # move an assembly from the SFP into the Core
                        assemType = None
                        locs = chain
                        if len(locs) < 2:
                            raise InputError("cascade starting with SFP must include a destination location")
                    else:
                        # move an assembly around the Core
```

Reply:
At your discretion. I just thought a comment in this if-statement would make it easier to read. Feel free to phrase it any way you want.